### DISCUSSION TITLE
Norton flagging and quarantining 

### DISCUSSION THREAD

**Discussion by rmburkhead** [d2phap/ImageGlass#145] [General] [answered]
Norton began flagging ImageGlass as suspicious this morning, quarantining the igcmd.exe. The threat it is assigning is Heur.AdvML.B which is a heuristic guess that it is a threat.  It is also quarantining the downloaded installation file.

**Comment by d2phap** [comment, marked as answer]
Hi @rmburkhead 

This may be a false positive detection. It's similar to https://github.com/d2phap/ImageGlass/issues/70 

igcmd.exe is to run some tasks of ImageGlass, such as installing theme, checking update, ...
You can check here: false positive

Phap
